README.md

```diff
@@ -1,19 +1,15 @@
-# Stytch Next.js example application
+# Stytch Next.js Page Router example application
 
 <p align="center">
   <img src="https://user-images.githubusercontent.com/100632220/217049841-b9eeb72a-3e50-4074-839a-e64ee5d4a88c.png" width="750">
 </p>
 
 ## Overview
 
-This example application demonstrates how one may use Stytch within a Next.js application. This project was bootstrapped with [Create Next App](https://nextjs.org/docs/api-reference/create-next-app).
-
-This project uses Stytch's [Next.js SDK](https://stytch.com/docs/sdks/javascript-sdk) which provides pre-built UI components, useful React hooks, headless methods to securely interact with Stytch, and is SSR friendly. This project also utilizes Stytch's [Node Backend SDK](https://www.npmjs.com/package/stytch) for authenticating the logged in user's session.  
```

Comment:
Why remove this? Not necessarily against it, but it seems like potentially helpful context

Reply:
The create next app portion? I think the page vs app router is imo a bit more important.

Reply:
Oh, I think you may mean the paragraph below. Agreed, will add back in.